CraftingRevisions/CraftingMenu/CraftingMenuPatches.cs

```diff
@@ -70,10 +72,10 @@ private static void Postfix(Panel_Crafting __instance)
 		[HarmonyPatch(typeof(CraftingRequirementQuantitySelect), "Enable")]
```

Comment:
No action required right now. However, I'd like to note that this patch might not be required anymore. It is purely a bugfix for Hinterland code.